src/components/SearchAlert.js

```diff
@@ -0,0 +1,10 @@
+import React, {Component} from 'react'
+export default class SearchAlert extends Component{
+     render(){ 
+        return (
+            <div className="alert alert-primary" >
+                Search cannot be empty
+            </div>
+        )
+    }
+}
```

Comment:
This should really be a functional component as it doesn't have any state or make use of any lifecycles.